src/components/network/network-explorer.js

```diff
@@ -199,9 +198,8 @@ const NetworkExplorer = ({
 
     function onDisplaySubstationFocusHandler(event, substation) {
         event.stopPropagation();
-        if (onSubstationFocus !== null) {
-            onSubstationFocus(substation.id);
-        }
+        console.info('jbo zut', substation);
```

Comment:
To remove

Reply:
done, but PR still in WIP as it need some change in commons ui and though migration to MUI V5